### DISCUSSION TITLE
Next.js App Router server crashes after a short time - warn The server is running out of memory, restarting to free up memory.

### DISCUSSION THREAD

**Discussion by MaximilianGaedke** [vercel/next.js#55471] [App Router]
Hi there,
I've been using the app router for a short time now and regularly, I would say every 7-15 minutes, the dev server crashes due to too less memory and returns a 500 error.
error in console: - warn The server is running out of memory, restarting to free up memory.
It can hardly be my Mac. And not about Next 13 itself, because I used Next 13 with the Pages Directory before and never had any problems with it.
In addition, the server takes 2-4 minutes to start again every time, which is also quite annoying.

Does anyone know how I can solve the two problems?

Thanks in advance!

**Comment by DIEGOHORVATTI** [comment]
#54359

**Comment by karkiahmedraid** [comment]
same problem i am facing
